tsconfig.json

```diff
@@ -0,0 +1,40 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "strict": true,
+    "jsx": "preserve",
+    "importHelpers": true,
+    "moduleResolution": "node",
+    "experimentalDecorators": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "sourceMap": true,
```

Comment:
The usage of `sourceMap` for debugging should be documented in the readme or wiki.

Reply:
included it in readme, not sure if that was what you were looking for.

Reply:
`sourceMap` allows us to debug using the browser dev tools https://www.youtube.com/watch?v=1ZmOTPeZQTM